src/bin/check.ts

```diff
@@ -28,20 +25,21 @@ export async function check(options: Options) {
     process.exit(1);
   }
   const config = loadBaseConfig(options);
-  for (const filePath of options._) {
-    if (!fs.existsSync(filePath)) {
+  const checkFile = async (filePath: string) => {
+    if (!(await fs.pathExists(filePath))) {
       STATS.otherIssues++;
```

Comment:
None, and will get faster speed

Reply:
Here I'm asking a few questions:
* Is `Promise.all` implemented via threads or coroutine?
* If it's threads, is `++` atomic for `number` (I assume it's the type of `otherIssues`)?

Reply:
Promise.all is just a wrpper of promise api, it's implemented in js main thread. You can check here for example implementation https://github.com/tjx666/js-handwriting-2022/blob/master/src/promise/promiseAll.js

Reply:
for await is bad performance because all the async function is executed one by one, but promise. all is let them all run currently, but waiting for promises completed

Reply:
check more here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all